devexp_demos/case_study/memory_leaks/images_1/web/README.md

```diff
@@ -31,27 +31,27 @@ After DevTools has connected to the running Flutter application, click on the Me
 
 Click on the leaky Image Viewer app (the image) and drag down for a few images to load.  The Memory profile chart should appear like the below chart.
 
-<img src="readme_images/memory_startup.png" />
+<img src="../readme_images/memory_startup.png" />
```

Comment:
I suspect the real fix here is to duplicate the `readme_images` directory in the `web` directory so it will be visible when the web directory is served by a web server. 
PTAL @kenzieschmoll